docs/website/docs/developers/general/developer-overview.md

```diff
@@ -258,6 +260,17 @@ should be in the format `[shape]xtype=[value]`. For example:
 1x5x3x1xf32=15,14,13,12,11,10,9,8,7,6,5,4,3,2,1
 ```
 
+Read from a numpy file:
+``` text
+--input=@input.npy
+```
+
+Read from a binary file with passing the shape and dtype because raw binary
+files have no metadata:
+``` text
+--input=[shape]xdtype=@input.bin
+```
```

Comment:
lint check wants blank lines around fenced code blocks
https://github.com/iree-org/iree/actions/runs/15766696478/job/44444406005?pr=21143#step:4:182
(I'd recommend installing pre-commit to just always run it automatically on each of your commits)
````suggestion
Read from a numpy file:

``` text
--input=@input.npy
```

Read from a binary file with passing the shape and dtype because raw binary
files have no metadata:

``` text
--input=[shape]xdtype=@input.bin
```
````